plugin/pkg/scheduler/schedulercache/interface.go

```diff
@@ -61,6 +61,9 @@ type Cache interface {
 	// After expiration, its information would be subtracted.
 	AssumePod(pod *api.Pod) error
 
+	// ForgetPod removes an assumed pod from cache.
+	ForgetPod(pod *api.Pod) error
```

Comment:
probably we need to also update the graph line34-46. for the assumed ones, now we can forget it?